src/module/actor/SoakFlow.ts

```diff
@@ -55,11 +56,45 @@ export class SoakFlow {
         modified = SoakRules.reduceDamage(actor, modified, roll.hits).modified;
         const incAndModDamage = {incoming, modified};
 
-        await createRollChatMessage({title, roll, actor, damage: incAndModDamage});
+        const options = {title, roll, actor, damage: incAndModDamage};
+        if (this.knocksDown(modified, actor)) {
+            options["knockedDown"] = true;
+        }
+        await createRollChatMessage(options);
 
         return roll;
     }
 
+    private knocksDown(damage: DamageData, actor:SR5Actor) {
+        // TODO: SR5 195 Called Shot Knock Down (Melee Only), requires attacker STR and actually announcing that called shot.
+        const gelRoundsEffect = this.isDamageFromGelRounds(damage) ? -2 : 0;  // SR5 434
+        const impactDispersionEffect = this.isDamageFromImpactDispersion(damage) ? -2 : 0  // FA 52
+        const limit = actor.getLimit('physical');
+        const effectiveLimit = limit.value + gelRoundsEffect + impactDispersionEffect
+        // SR5 194
+        return damage.value > effectiveLimit || damage.value >= 10;
+    }
+
+    private isDamageFromGelRounds(damage: DamageData) {
+        if (damage.source && damage.source.actorId && damage.source.itemId) {
+            const attacker = game.actors.find(actor => actor.id == damage.source?.actorId);
+            if (attacker) {
+                const item = attacker.items.find(item => item.id == damage.source?.itemId);
```

Comment:
Foundry 0.8 has a simple DocumentCollection get helper for this.
attacker.items.get(damage.source.itemId);